### PR TITLE
OCPBUGS-13926: change the operator log level to default normal in the deployment

### PR DESCRIPTION
**Pull request by lance5890**
[issue 290](https://github.com/openshift/cluster-openshift-controller-manager-operator/issues/290)

### CODE REVIEW

**Comment by lance5890** [comment]
@atiratree

File: manifests/09_deployment.yaml [hunk 1]
```diff
@@ -43,7 +43,6 @@ spec:
         command: ["cluster-openshift-controller-manager-operator", "operator"]
         args:
         - "--config=/var/run/configmaps/config/config.yaml"
-        - "-v=4"
```
Comment (atiratree) [inline manifests/09_deployment.yaml:46]: It should not be important what is set here, but on a closer look it is! It seems OCM-o does not support obtaining verbosity through `OpenShiftControllerManager.operatorLogLevel` object. Seems like we have `-v=4` as a hack to have at least middle ground of logging here (`DEBUG`), so we can proceed with removal of this line.

OAS-o uses OperatorLoggingController here https://github.com/openshift/cluster-openshift-apiserver-operator/blob/eaaa4e6ce3e9ff1d2c4594406741ac10e98cb219/pkg/operator/starter.go#L290. We should do something similar in KAS-o before we remove this line.

Comment (lance5890) [reply manifests/09_deployment.yaml:46]: i will get a closer look too

Comment (atiratree) [reply manifests/09_deployment.yaml:46]: Yes OCM pods restart and seem to react to `logLevel` correctly. But changing the `operatorLogLevel` does not lead to change in OCM-o pod behaviour or a restart in my observation.

Comment (lance5890) [reply manifests/09_deployment.yaml:46]: > Yes OCM pods restart and seem to react to `logLevel` correctly. But changing the `operatorLogLevel` does not lead to change in OCM-o pod behaviour or a restart in my observation.

I think this is ok, change the operatorLogLevel will only restart the OCM pods, but not the operator itself

Comment (atiratree) [reply manifests/09_deployment.yaml:46]: yes no restart is correct and expected, nevertheless the log level change should occur in OCM-o pod

Comment (atiratree) [reply manifests/09_deployment.yaml:46]: we first have to fix the logging feature of the operator before we can remove this line

Comment (lance5890) [reply manifests/09_deployment.yaml:46]: > we first have to fix the logging feature of the operator before we can remove this line

I have commit a PR to fix the logging feature of the operator PR 292[](https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292)

Comment (lance5890) [reply manifests/09_deployment.yaml:46]: with the https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292 PR has merged

**Comment by lance5890** [comment]
@atiratree @ingvagabund  with the https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292 has merged，this PR should also be merged .  wdyt  !

**Comment by atiratree** [comment]
+1, this field should not have any effect anymore
/ok-to-test

**Comment by atiratree** [comment]
/retitle OCPBUGS-13926: change the operator log level to default normal in the deployment

**Comment by atiratree** [comment]
relevant to https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292
/assign @ingvagabund

**Comment by lance5890** [comment]
> relevant to #292 /assign @ingvagabund

@ingvagabund wdyt

**Comment by lance5890** [comment]
As the "-v=4" option doesn't work anymore, we shoule remove this option

**Comment by ingvagabund** [comment]
Checking https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-openshift-controller-manager-operator/292/pull-ci-openshift-cluster-openshift-controller-manager-operator-master-e2e-aws-operator/1664191374691405824/artifacts/e2e-aws-operator/gather-extra/artifacts/events.json of the original https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292 there's the following event:
```
{
            "apiVersion": "v1",
            "count": 1,
            "eventTime": null,
            "firstTimestamp": "2023-06-01T09:05:07Z",
            "involvedObject": {
                "apiVersion": "apps/v1",
                "kind": "Deployment",
                "name": "openshift-controller-manager-operator",
                "namespace": "openshift-controller-manager-operator",
                "uid": "bddedbef-5a2e-46b7-85d6-cbb3b75ee59c"
            },
            "kind": "Event",
            "lastTimestamp": "2023-06-01T09:05:07Z",
            "message": "Operator log level changed from \"Debug\" to \"Normal\"",
            "metadata": {
                "creationTimestamp": "2023-06-01T09:05:07Z",
                "name": "openshift-controller-manager-operator.17647da4ccfc3f55",
                "namespace": "openshift-controller-manager-operator",
                "resourceVersion": "4730",
                "uid": "bdfce41b-5bc2-4a9f-bc21-8121db79a44d"
            },
            "reason": "OperatorLogLevelChange",
            "reportingComponent": "",
            "reportingInstance": "",
            "source": {
                "component": "openshift-controller-manager-operator-loggingsyncer"
            },
            "type": "Normal"
        },
```
which proves the log level changed to normal/2. Checking events in the CI job of this PR I don't see any since the default log level is normal now.

/approve

**Pull request merged**